app/scripts/modules/kubernetes/pipeline/stages/runJob/runJobStage.js

```diff
@@ -55,6 +57,9 @@ module.exports = angular.module('spinnaker.kubernetes.pipeline.stage.runJobStage
       _.set(this.stage, 'container.imageDescription.fromTrigger', false);
     }
 
+    if (!_.has(this.stage, 'dnsPolicy')) {
```

Comment:
If we're sure `this.stage` exists, it's cleaner to omit the `has`/`set` here. 
e.g.,
```
if (!this.stage.dnsPolicy) {
  this.stage.dnsPolicy = 'ClusterFirst';
}
```